pkg/venafi/cloud/connector.go

```diff
@@ -830,6 +830,10 @@ func getCloudRequest(c *Connector, req *certificate.Request) (*certificateReques
 		cloudReq.ValidityPeriod = validityHoursStr
 	}
 
+	if req.ValidityDuration != nil {
+		cloudReq.ValidityPeriod = "PT" + strings.ToUpper((*req.ValidityDuration).Truncate(time.Second).String())
+	}
+
```

Comment:
Do youknow whether VaaS can parse embedded duration suffixes (`1H1M1S`) or whether it expects only a single suffix (`90060S`) ?

Reply:
The 1 hour example that we use in our tests works.

Reply:
But that would be rendered as `1H`. I'm asking whether VaaS will be able to parse durations like `1M1S` which contain both `M` and `S`  or whether it would require `61S`?

Reply:
I did some more testing & this seems to work correctly.